pom.xml

```
@@ -600,7 +600,7 @@ under the License.
         <!-- Use Scala 2.13 by default -->
         <scala.version>2.13.14</scala.version>
         <scala.binary.version>2.13</scala.binary.version>
-        <spark.version>4.0.0-preview1</spark.version>
+        <spark.version>4.0.0</spark.version>
         <spark.version.short>4.0</spark.version.short>
         <parquet.version>1.13.1</parquet.version>
```

Parquet was upgraded in the 4.0.0 release version:
```suggestion
        <parquet.version>1.15.2</parquet.version>
```